launch/preprocess.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
+  <param name="IMAGE_INPUT_RAW" value="/camera/image_raw" type="str" />
```

Comment:
Lets not capitalize all our parameters. They should look the same as the topic names